### DISCUSSION TITLE
Does anyone have any informations about the cookie named “noDeprecation” ?

### DISCUSSION THREAD

**Discussion by s4a-aandriamanga** [vercel/next.js#71621] [Help]
### Summary

I'm looking for an explanation if anyone knows more about the role of the cookie named "noDeprecation" and where it's come from.

Maybe set by default by NextJS ?

Because, our GDPR compliance bot validator who checks if cookies is set without user consent, sends us back that this cookie named "noDeprecation" with expiration value "-1" as a 1st part cookie.

But we never created it on our side. :(

Thanks a lot for your answer.

**Comment by icyJoseph** [comment]
I can't find anything regarding this on the Next.js codebase.

Where is your project hosted?

**Reply by s4a-aandriamanga** [reply]
My project is hosted by GCP on K8S pods.

It could be related ?

**Comment by s4a-aandriamanga** [comment]
@leerob do you know if there is a possibility that NextJS sets this cookie named `noDeprecation` ?

**Reply by icyJoseph** [reply]
https://github.com/search?q=repo%3Avercel%2Fnext.js%20noDeprecation&type=code zero results, other than your own posts :/

**Reply by icyJoseph** [reply]
Btw, is this set on your own domain? Is it a JavaScript available cookie, aka Httponly?

**Reply by s4a-aandriamanga** [reply]
@icyJoseph yes it's set on my own domain.

**Reply by icyJoseph** [reply]
And, is it a Httponly cookie?

**Reply by icyJoseph** [reply]
Do you use any libraries in Middleware? I couldn't really find anything meaningful in GitHub, trying to set such a cookie 🤔

**Reply by s4a-aandriamanga** [reply]
We use `next-intl` (https://next-intl-docs.vercel.app) in middleware

**Reply by icyJoseph** [reply]
Well this is a dead end... Is this website public? Which flows set the cookie?
